src/styles/_mixins.scss

```diff
@@ -94,19 +94,28 @@
   flex-wrap: wrap;
 }
 
+@mixin language-button-hover($text) {
```

Comment:
Should it be `language-button-hover` or `language-button--hover`?

Reply:
Updated to `language-button--hover`, and also changed `link-size-expanded` to `link-size--expanded` in styling updates 2 PR